scripts/create_test_customer.py

```diff
@@ -72,10 +67,10 @@ def __init__(self, conf):
         self.customer = self._config.get('customer')
         self.koku = self._config.get('koku')
 
-        self.endpoint_base = 'http://{}:{}/{}/v1/'.format(
+        self.endpoint_base = 'http://{}:{}{}/v1/'.format(
             self.koku.get("host"),
             self.koku.get("port"),
-            self.koku.get("prefix"))
+            self._config.get("api_prefix") or self.koku.get("prefix"))
```

Comment:
@blentz How about this?

Reply:
This looks really good. It's definitely on the right track.

My goal is to have both the `argparse` config and the loaded YAML populate a single configuration object. That way, we don't have inconsistencies in picking which value to use. Additionally, a CLI arg can then override a defined YAML value, if needed.